pkg/uprobetracer/tracer.go

```diff
@@ -230,12 +244,15 @@ func (t *Tracer[Event]) attach(containerPid uint32) {
 			continue
 		}
 
-		t.logger.Debugf("attaching uprobe %q to container %d: %q", t.attachSymbol, containerPid, filePath)
+		t.logger.Debugf("attaching uprobe %q to container %d: %q", t.progName, containerPid, filePath)
 		attachedUUIDs = append(attachedUUIDs, fileUUID)
 
 		inode, exists := t.inodeRefCount[fileUUID]
 		if !exists {
-			progLink, _ := t.attachUprobe(file)
+			progLink, err := t.attachUprobe(file)
+			if err != nil {
+				t.logger.Debugf("failed to attach uprobe %q: %q", t.progName, err.Error())
```

Comment:
But using `Errorf` will change the log level to `error`? Failure while attaching to uprobes are common, because the target files may not have the required symbols, and in such cases we just ignore them. If the user want to know why some files are not attached, he can enable the verbose mode.

Reply:
Ok, please keep Debugf then :)

Even with Debugf, `%q..., err.Error()` should be changed to `%w..., err`.

Reply:
Seems not, my Goland IDE says "This verb can be used only in 'fmt.Errorf' calls (%w) ". 
And the output with `%w` is:
DEBU[0000] failed to attach uprobe "trace_uprobe_free": %!w(*errors.errorString=&{123})

Reply:
Oops, sorry for the wrong comment.